device/iot-device-samples/android-sample/app/src/main/java/com/iothub/azure/microsoft/com/androidsample/MainActivity.java

```diff
@@ -20,6 +20,9 @@
 public class MainActivity extends AppCompatActivity {
 
     String connString = "[device connection string]";
+    String deviceId = "MyAndroidDevice";
```

Comment:
Why is this deviceID needed ? I don't see it being needed in anything other than msgStr. If you really intend to use it there could you rename it to something else ? like deviceName ? Because this can be easily confused with deviceID we use for this device.